ush/wave_grib2_sbs.sh

```diff
@@ -71,8 +71,18 @@ ENSTAG=""
 if [[ -n ${waveMEMB} ]]; then ENSTAG=".${membTAG}${waveMEMB}" ; fi
 outfile="${WAV_MOD_TAG}.${cycle}${ENSTAG}.${grdnam}.${grdres}.f${FH3}.grib2"
 
+#create the COM directory var
+com_dir="COMOUT_WAVE_GRID_${grdnam}_${grdres}"
```

Comment:
Recommend doing the indirection once, then just using a regular variable later
```suggestion
com_varname="COMOUT_WAVE_GRID_${grdnam}_${grdres}"
com_dir="${!com_varname}"
```

Then all the `${!com_dir}` later on just become `${com_dir}`